ci: add ci bot for auto assign issue (#10511)
rook/rook: Documentation/Contributing/development-flow.md

Now you should have at least `origin` and `upstream` remotes. You can also add other remotes to collaborate with other contributors.

### Auto assign Issue

> You can auto-assign any issue that is been created in Rook Upstream repo, by commenting

Reviewer: ```suggestion
You can self-assign any issue that is been created in Rook upstream repo by adding
```

> `\assign` in as a issue comment.

Reviewer: Does it just need to contain, or does it need to be the only text in the comment?
```suggestion
an issue comment with `/assign` in the body.
```

> You can self-assign any issue that is been created in Rook upstream repo, by adding
> an issue comment with `/assign` in the body.

Reviewer 2: ```suggestion
You can self-assign any issue that is open and not assigned to anyone in Rook upstream repo, by adding a comment with `/assign` in the body.
```

Author: Done changed the `not assigned` thing that will be really helpful

But `an issue comment` seems fine and more understandable so keep like that, 
let me if it doesn't look good.
